mirage/factories/utils.ts

```diff
@@ -1,12 +1,27 @@
-import { faker } from 'ember-cli-mirage';
+import { faker, ModelInstance, Server } from 'ember-cli-mirage';
+import SeedRandom from 'seedrandom';
 
 import { RegistrationMetadata, Schema } from 'ember-osf-web/models/registration-schema';
 
-export const guid = (id: number, type: string): string => {
-    const numPart = String(id);
-    const typPart = type.substr(0, 5 - numPart.length);
-    return `${typPart}${numPart}`;
-};
+const GUID_CHARS = 'abcdefghijklmnopqrstuvwxyz0123456789';
+
+export function guid(referentType: string) {
+    return (id: number) => {
+        // Generate a pseudo-random guid
+        const prng = new SeedRandom(`${referentType}-${id}`);
```

Comment:
Any reason to not use fakers random number generator here? Looks like you can seed it as well, albeit with a number.

Reply:
Oh, that probably makes more sense. It looks like faker only allows global seeding, which is mildly annoying, but seems better than adding a dependency.